packages/api/src/util/ensemble.js

```diff
@@ -61,6 +61,180 @@ const ENSEMBLE_UNITS_MAPPING = {
   },
 };
 
+/**
+ * Retrieves a valid Ensemble organisation by its UUID.
+ * @param {uuid} org_uuid
+ * @returns {Object} - The organisation object.
+ * @throws {Object} - Throws an error with status 404 if the organisation is not found within the Ensemble organisations we have access to
+ * @async
+ */
+const getValidEnsembleOrg = async (org_uuid) => {
+  const allRegisteredOrganisations = await getEnsembleOrganisations();
+
+  const organisation = allRegisteredOrganisations.find(({ uuid }) => uuid === org_uuid);
```

Comment:
Teeny thing if you end up making more commits.. could probably return here without const and rename function away from "valid"